hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlConnector.java

```diff
@@ -372,6 +380,43 @@ public Vertex deleteProcessor(@Nonnull DAG dag, @Nonnull Table table0) {
         );
     }
 
+    // Returns if upsert is supported for the given dialect
+    protected boolean isUpsertSupported(JdbcTable jdbcTable) {
+        boolean result = false;
+        SqlDialect dialect = jdbcTable.sqlDialect();
+        if (dialect instanceof MysqlSqlDialect ||
+            dialect instanceof PostgresqlSqlDialect ||
+            dialect instanceof H2SqlDialect) {
+            result = true;
+        }
+        return result;
```

Comment:
```suggestion
        SqlDialect dialect = jdbcTable.sqlDialect();
        return dialect instanceof MysqlSqlDialect ||
            dialect instanceof PostgresqlSqlDialect ||
            dialect instanceof H2SqlDialect;
```
?

Reply:
Ok and moved the code to UpsertBuilder. Since it is more relevant in that class